code/modules/surgery/surgery.dm

```diff
@@ -43,7 +43,7 @@
 	if(replaced_by == /datum/surgery)
 		return FALSE
 
-	if(HAS_TRAIT(user, TRAIT_SURGEON))
+	if(HAS_TRAIT(user, TRAIT_SURGEON) || istype(user) && user.mind && HAS_TRAIT(user.mind, TRAIT_SURGEON))
```

Comment:
I feel like you should just add trait to person not mind when creating abductors instead of doing whatever is this

Reply:
this ^

Reply:
Its how the abductors did it, iam not sure if it had to be that way so, I will edit that then

Reply:
Lastest PR should fix it

Reply:
No, you got it wrong. Traits should be added to bodies not minds.

Reply:
It makes more sense to me that the abductor scientist KNOWS how to do the surgery instead of his species allow it to be that way. How to use the baton and such is also stored in the mind because he KNOWS it